apps/addons/models.py

```diff
@@ -496,7 +496,9 @@ def delete(self, msg='', reason=''):
                         _current_version=None, guid=None)
             models.signals.post_delete.send(sender=Addon, instance=self)
 
-            send_mail(subject, email_msg, recipient_list=to)
+            # Don't localize email to admins, use 'en-US' always.
+            with user.activate_lang('en-US'):
+                send_mail(subject, email_msg, recipient_list=to)
```

Comment:
I don't think you want to do that, but rather something like https://github.com/mozilla/olympia/blob/master/apps/editors/tasks.py#L60 (use https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.translation.override)